lib/connectors/memory.js

```diff
@@ -823,7 +824,7 @@ Memory.prototype.replaceOrCreate = function(model, data, options, callback) {
     // Calling _createSync to update the collection in a sync way and
     // to guarantee to create it in the same turn of even loop
     return self._createSync(model, data, function(err, id) {
-      if (err) return process.nextTick(function() { cb(err); });
+      if (err) return process.nextTick(function() { callback(err); });
```

Comment:
@bajtos @superkhau either one is fine with me; consistency is what I would follow... I did a quick search in `lib/connectors/memory.js` and it seems most of the methods use `callback`, so I would personally go for `callback` in this case; however I agree if I had the option with not lots of `callback`s in the file, I would go for `cb` rather than `callback` but it is a nitpick.

@superkhau this one actually was a bug since there was no `cb` in this method and that's why @bajtos changed it to `callback`.